cob_bms_driver/src/power_state_aggregator.py

```diff
@@ -63,11 +71,26 @@ def calculate_power_consumption(self):
 
     def calculate_relative_remaining_capacity(self):
         if self.full_charge_capacity != None and self.remaining_capacity != None:
-            return round(100.0*(self.remaining_capacity/self.full_charge_capacity), 3)
+            if self.voltage <= self.threshold_voltage_critical and self.current > -self.steady_current:
+                self.relative_remaining_capacity = 5 # % of full capacity
+            elif self.voltage <= self.threshold_voltage_error and self.current > -self.steady_current:
+                self.relative_remaining_capacity = 10 # % of full capacity
+            elif self.voltage <= self.threshold_voltage_warn and self.current > -self.steady_current:
+                self.relative_remaining_capacity = 15 # % of full capacity
+            elif self.voltage <= self.threshold_voltage_mid and self.current > -self.steady_current:
```

Comment:
because the current drawn from the battery is negative. I find it unintuitve to edit a negative steady current in the yaml